paimon-common/src/main/java/org/apache/paimon/io/cache/CacheManager.java

```diff
@@ -37,69 +33,46 @@ public class CacheManager {
      */
     public static final int REFRESH_COUNT = 10;
 
-    private final Cache<CacheKey, CacheValue> cache;
+    private final InternalCache internalCache;
 
     private int fileReadCount;
 
     public CacheManager(MemorySize maxMemorySize) {
-        this.cache =
-                Caffeine.newBuilder()
-                        .weigher(this::weigh)
-                        .maximumWeight(maxMemorySize.getBytes())
-                        .removalListener(this::onRemoval)
-                        .executor(Runnable::run)
-                        .build();
+        this(InternalCache.CacheType.CAFFEINE, maxMemorySize);
```

Comment:
When use guava cache? If java 8, how control ?

Reply:
We will introduce an option for the cache type in the next PR, and only guava cache can be used in the sort file store